agimus_controller_ros/hpp_agimus_controller.py

```diff
@@ -88,7 +90,8 @@ def plan_and_first_solve(self):
 
         # Plan
         q_init = [*sensor_msg.joint_state.position]
-        self.ps = self.hpp_interface.set_panda_planning(q_init, self.q_goal)
+        self.hpp_interface.set_panda_planning(q_init, self.q_goal)
+        self.ps = self.hpp_interface.ps
```

Comment:
We should not need the ps object in this class.

Reply:
it's used to retrieve the hpp path in the function `plan_and_first_solve`, that's the only time, it indeed doesn't have to be an attribute of the class